packages/slate-hyperscript/src/index.js

```diff
@@ -202,7 +202,7 @@ const CREATORS = {
     }
 
     if (!isEmpty(props)) {
-      selection = selection.merge(props).normalize(document)
+      selection = selection.loadProps(props).normalize(document)
```

Comment:
Why do we call this `loadProps`? Why not just overload `merge` if we need to do custom logic? I assume it's for cases like:

```js
const next = range.merge({ anchorKey: '1' })
```

I think I'd rather overload `merge` for now with deprecation warnings when those key names are used, and prevent updating by those key names in the future.